client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -80,7 +80,7 @@ module.exports = React.createClass( {
 		} else if ( getIncludedDomain( this.props.cartItem ) ) {
 			info = getIncludedDomain( this.props.cartItem );
 		} else {
-			info = domain;
+		    info = this.props.cartItem.is_domain_registration ? domain : this.props.selectedSite.domain;
```

Comment:
@alleh: Many thanks for looking at this issue :)

We have several conditions here and it would be great to avoid nested conditions here like:
```js
} else if ( this.props.cartItem.is_domain_registration ) {
	info = domain;
} else {
	info = this.props.selectedSite.domain;
}  
```

BTW, we indent with tabs:
> Indentation with tabs.

This is from our [JavaScript Coding Guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md). You can check this document to find more details.